node/send_receive/package.json

```diff
@@ -29,7 +32,7 @@
   },
   "homepage": "https://github.com/Azure/azure-event-hubs#readme",
   "scripts": {
-    "lint": "jshint --show-non-errors .",
+    "lint": "jshint --show-non-errors . && tslint -c ./tslint.json \\\"./lib/**/*.ts\\\"",
```

Comment:
I'd rather keep the typescript stuff separate. If someone is contributing to the JS side and not using typescript at all, let's not force him to install the whole typescript environment.